src/Core_Sideloader_Studio/UniversalAutoResolver/Core.UAR.Studio.cs

```diff
@@ -401,6 +414,8 @@ internal static void ResolveStudioRamp(ExtensibleSaveFormat.PluginData extData,
             if (extData != null && extData.data.ContainsKey("rampInfoGUID"))
             {
                 string rampGUID = (string)extData.data["rampInfoGUID"];
+                string rampAuthor = (string)extData.data["rampInfoAuthor"];
+                string rampWebsite = (string)extData.data["rampInfoWebsite"];
```

Comment:
Oops, one of these days I'll remember dictionaries store nulls and therefore throw exceptions on not found instead of return null...but not this day...